tests/cli/configs/test_postgres.py

```diff
@@ -23,7 +28,40 @@ def test_postgres_target_configs_get_configs():
         "user": "prefect",
         "password": "prefect_password",
         "host": "host",
-        "port": 8080,
+        "port": "8080",
+    }
+    for k, v in actual.items():
+        actual_v = v.get_secret_value() if isinstance(v, SecretStr) else v
+        expected_v = expected[k]
+        assert actual_v == expected_v
+
+
+def test_postgres_target_configs_get_configs_for_sqlalchemy_connector():
+    configs = PostgresTargetConfigs(
+        credentials=SqlAlchemyConnector(
+            connection_info=ConnectionComponents(
```

Comment:
Can you add additional config that shouldn't show up in the result to ensure that it is filtered out?